packages/desktop-client/src/components/mobile/MobileAmountInput.js

```diff
@@ -355,14 +332,7 @@ class FocusableAmountInput_ extends PureComponent {
 }
 
 export const FocusableAmountInput = forwardRef((props, ref) => {
-  return (
-    <AmountAccessoryContext.Consumer>
-      {context => (
-        // eslint-disable-next-line
-        <FocusableAmountInput_ ref={ref} {...props} context={context} />
-      )}
-    </AmountAccessoryContext.Consumer>
-  );
+  return <FocusableAmountInputInner ref={ref} {...props} />;
```

Comment:
Since this is just assigning the ref to the out component, I think you can eliminate the wrapper?

Reply:
We can! Done in https://github.com/actualbudget/actual/pull/1340/commits/0c7a57efec17e8cf92be2a8473a1de45fab1bd89